README.md

```diff
@@ -11,40 +11,40 @@
 
 Released: June 2, 2020</p>
 
-Written by [Barak Naveh](mailto:barak_naveh@users.sourceforge.net)  and Contributors
+Written by [Barak Naveh](mailto:barak_naveh@users.sourceforge.net) and Contributors
 
 (C) Copyright 2003-2020, by Barak Naveh and Contributors. All rights
 reserved.
 
 Please address all contributions, suggestions, and inquiries to the [user mailing list](https://lists.sourceforge.net/lists/listinfo/jgrapht-users)
 
-## Introduction ##
+## Introduction
 
 JGraphT is a free Java class library that provides mathematical graph-theory objects and algorithms. It runs on Java 2 Platform (requires JDK 1.8 or later starting with JGraphT 1.0.0, and JDK 11 or later starting with JGraphT 1.4.1).
 
 JGraphT may be used under the terms of either the
 
  * GNU Lesser General Public License (LGPL) 2.1
-   http://www.gnu.org/licenses/lgpl-2.1.html
+   <http://www.gnu.org/licenses/lgpl-2.1.html>
 
 or the
 
  * Eclipse Public License (EPL)
-   http://www.eclipse.org/org/documents/epl-v20.php
+   >http://www.eclipse.org/org/documents/epl-v20.php>
```

Comment:
The first one should be less-than, not greater-than, right?

Reply:
Oh, yeah, sorry for that!